Xamarin.PropertyEditing.Mac/Controls/StringEditorControl.cs

```diff
@@ -120,7 +122,7 @@ protected override void UpdateErrorsDisplayed (IEnumerable errors)
 
 		protected override void SetEnabled ()
 		{
-			this.stringEditor.Editable = ViewModel.Property.CanWrite;
+			this.stringEditor.Editable = this.stringEditor.Enabled = ViewModel.Property.CanWrite;
```

Comment:
The Enabled property with False value makes the field **not focusable** and therefore not editable, maybe then we can remove it.

Reply:
If we remove the Enabled then we don't get the text changing to light gray automagically. Which means we'd need to create a custom TextField, to change the TextColour depending on Dark/Light mode when enabled or disabled.
What scenario do you feel needs you to be able to focus a disabled control?

Note: None of the numeric editors allow focusing when they are disabled, nor the drop-downs or popups.

Reply:
Also Xcode behaves the same way. If a `NSTextField` is disabled, when tabbing it will be skipped. So I think this is expected behaviour on Mac.

Reply:
I mean remove the Editable property set, because with a Enabled field in disabled user cannot edit

Reply:
Yes. Thanks. Fixed.